FF1Lib/RomStampModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Schema;
+using System.IO;
+
+namespace FF1Lib
+{
+	public class RomStampModel
+	{
+		public String s { get; set; }
```

Comment:
Now THIS is where comments would be useful.

For example, what each of these letters means, and why they're just one letter (to minimize the JSON size in the ROM).